packages/@okta/vuepress-site/docs/guides/terraform-manage-multiple-domains/main/index.md

```diff
@@ -0,0 +1,262 @@
+
+1. Log into your domain provider for your custom domain and navigate to the domain record editor.
+
+1. Create a `CNAME` record that maps your custom brand to a domain that Okta creates just for this custom domain. The values you need for this step are in the object in the JSON output with a `record_type` of `"CNAME"`.
+
+    1. Set the fully qualified domain name (FQDN) of the `CNAME` record to the value of the `fqdn` field in the JSON output.
+
+    1. Set the value of the `CNAME` record to the string in the `values` array of the JSON output. This is the name for your custom domain, such as `example-org.customdomains.oktapreview.com`.
+
+    1. Save the new record in your ISP's control panel for DNS records.
+
+1. Create a `TXT` record to support Okta domain verification. The values you need for this step are in the object in the JSON output with a `record_type` of `"TXT"`.
+
+    1. Set the fully qualified domain name (FQDN) of the `TXT` record to the value of the `fqdn` field in the JSON output. Include any underscores that are at the start of the value.
+
+    1. Set the value of the `TXT` record to the string in the `values` array of the JSON output. In the previous example of a JSON record, this value is `NNQY-CZHKKHLKJKZODDYZOT5ZV3a7gM8-bxHc9o`.
+
+    1. Save the new record in your ISP's control panel for DNS records.
+
+1. Wait until the DNS records propagate. This step may take up to twenty-four hours, though could take as little as 15 minutes. Check if your DNS records are available using a tool, such as [Dig](https://toolbox.googleapps.com/apps/dig/).
+
+### Request domain verification
+
+Request Okta domain verification once the DNS changes have propagated. Okta confirms that the `TXT` records for your domain contain the Okta-generated value proving that you control the domain. In addition, Okta checks the `CNAME` record if you're using an Okta-managed certificate.
+
+1. Add an [okta_domain_verification](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/domain_verification) Terraform resource to your configuration.
+
+1. Set the `domain_id` attribute to your domain's ID:
+
+    ```hcl
+    resource "okta_domain_verification" "verify_okta" {
+      domain_id = {okta_domain.brand1.id}
+    }
+    ```
+
+1. Use `terraform plan` and `terraform apply` to deploy the domain verification resource. Okta may take up to fifteen minutes to check your domain. If the `terraform apply` command succeeds, verification succeeded, and you can proceed to the next step. If the command fails, wait a few minutes and try again.
+
+### Confirm your custom domain triggers Okta sign-in
+
+
+Open your custom domain in your browser. The Okta sign-In page should appear. If the page doesn't appear, it may be because your machine or DNS providers are caching the old records. You can try flushing the DNS cache using a tool such as [Google DNS ](https://google-public-dns.appspot.com/cache) or [Open DNS](https://cachecheck.opendns.com/).
```

Comment:
```suggestion
Open your custom domain in your browser. The Okta sign-in page should appear. If the page doesn't appear, it may be because your machine or DNS providers are caching the old records. You can try flushing the DNS cache using a tool such as [Google DNS ](https://google-public-dns.appspot.com/cache) or [Open DNS](https://cachecheck.opendns.com/).
```

Reply:
Fixed in next commit